Automattic/wp-calypso
# Stepper: Replace domain upsell exitFlow() usage with stepper submit() function

Related to #

## Proposed Changes

* The changes in this PR remove the `exitFlow` function from the domain upsell flow and replaces it with the `submit` function. We should only navigate between screens/pages in the Stepper framework using the `submit` function.

## Testing Instructions



* Checkout this branch.
* Run `yarn start`.
* Create a new "free flow" launchpad enabled site or use an existing free flow site that hasn't been launched yet.
* Click the "Choose a domain" checklist task on the launchpad screen.
* Click the "Back" button on the top left side of the domains selection screen and verify it navigates back to the launchpad screen.

## Pre-merge Checklist
